src/server/page-renderer.js

```diff
@@ -76,7 +76,14 @@ class PageRenderer extends PageRendererBase {
       contentProps: props
     });
 
-    return res.type('html').send(html);
+    return res
+      .type('html')
+      .set({
+        'Cache-Control': 'max-age=0, no-cache, no-store, must-revalidate',
+        'Pragma': 'no-cache',
+        'Expires': 'Wed, 11 Jan 1984 05:00:00 GMT'
```

Comment:
Just to double check that this is not a copy paste thing: we need to set this in the past so that the cache is expired immediately, right?

Reply:
Yes :laughing: